pr-request-template.md

```diff
@@ -0,0 +1,77 @@
+# Pull Request Template
+
+Thank you for your contribution to Alumnium!
+Please fill out this template completely to help us understand and review your PR efficiently.
+
+## Summary
+What is this pull request about?
+Provide a brief description of what the PR does, the problem it solves, and any context around the implementation.
+
+## Motivation
+Why is this change being made?
+Describe the use case, problem, or opportunity that this PR addresses.
+Explain how this change contributes to the project's goals or improves functionality.
+
+## Related Issue(s)
+If this PR fixes or is related to an open issue, link it here.
+Example: Closes #123
+
+Closes #
+
+## Type of Change
+Mark the relevant options.
+
+- [ ] Bug fix  
+- [ ] New feature  
+- [ ] Documentation update  
+- [ ] Test coverage improvement  
+- [ ] Refactoring  
+- [ ] CI/CD enhancement
+- [ ] Breaking change
+- [ ] Other (please specify):
+
+## Code or UI Demos (if applicable)
+Add screenshots, logs, terminal output, or code snippets that help reviewers understand the change.
+Useful especially for UI changes or error fixes.
+
+## Testing Strategy
+How have you tested your changes?
+Mention tools used, test cases added or updated, and how the change was validated.
+
+## Local Testing
+Describe the environment and steps you used to test these changes locally.
+Include any relevant configuration details or test data.
+
+## Dependencies
+List any new dependencies added or existing ones modified by this PR.
+
+## Performance Considerations
+Describe any performance impacts this change might have, positive or negative.
+Include benchmark results if available.
+
+## Security Considerations
+Outline any security implications of these changes.
+Document how security concerns have been addressed if applicable.
+
+## Deployment Notes
+Include any special instructions required for deploying this change.
+Mention database migrations, environment variables, or other requirements.
```

Comment:
We don't have a database or migrations. GitHub handles the deployment, so if someone changes the config files, we'll see the required changes. I think this section can be skipped.